src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -856,6 +856,79 @@ std::set<std::vector<element::Type>> jit_subtract_emitter::get_supported_precisi
     return {{element::f32, element::f32}};
 }
 
+/// SWISH ///
+jit_swish_emitter::jit_swish_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                     dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                     const std::shared_ptr<ov::Node>& node)
+        : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+    sigmoid_emitter = std::make_unique<jit_sigmoid_emitter>(h, host_isa, node);
+}
+
+jit_swish_emitter::jit_swish_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                     dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                     const float beta,
+                                     const ov::element::Type exec_prc)
+        : jit_emitter(host, host_isa, exec_prc), beta(beta) {
+    prepare_table();
+    sigmoid_emitter = std::make_unique<jit_sigmoid_emitter>(h, host_isa, exec_prc);
+}
+
+size_t jit_swish_emitter::get_inputs_count() const {return 1; }
+
+size_t jit_swish_emitter::get_aux_vecs_count() const {
+    return sigmoid_emitter->get_aux_vecs_count() + 1;
+}
+
+size_t jit_swish_emitter::get_aux_gprs_count() const {
+    return sigmoid_emitter->get_aux_gprs_count() + 1;
+}
+
+void jit_swish_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OV_CPU_JIT_EMITTER_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+void jit_swish_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    OV_CPU_JIT_EMITTER_ASSERT(exec_prc_ == ov::element::f32, "unsupported precision: " + exec_prc_.to_string());
+
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    const TReg vmm_src(in_vec_idxs[0]);
+    const TReg vmm_dst(out_vec_idxs[0]);
+    const TReg vmm_aux0(aux_vec_idxs[sigmoid_emitter->get_aux_vecs_count()]);
+
+    // x*beta
+    h->ld1r(vmm_aux0.s, table_val2("beta"));
+    h->fmul(vmm_aux0.s, vmm_aux0.s, vmm_src.s);
+
+    // sigmoid(x*beta)
+    sigmoid_emitter->emit_code(
+            { vmm_aux0.getIdx() },
+            out_vec_idxs,
+            aux_vec_idxs,
+            aux_gpr_idxs);
+
+    // x*sigmoid(x*beta)
+    h->fmul(vmm_dst.s, vmm_dst.s, vmm_src.s);
```

Comment:
It will work incorrectly in case src and dst uses the same register

Reply:
fixed, thanks